aggregator/src/binaries/aggregator.rs

```diff
@@ -134,6 +160,7 @@ async fn run_aggregator(
     join!(
         aggregator_server,
         garbage_collector_future,
+        key_rotator_future,
         aggregator_api_future
     );
```

Comment:
This is an existing issue, but I think we should `tokio::spawn()` the four futures here, and `join!()` the resulting join handles instead. This would let us skip a bunch of bookkeeping in the join macro on each poll, and let the respective accept loops, timers, and other await points run concurrently with each other on separate threads.

Reply:
Simple enough change. Note that I've wrapped these in a `try_join!` since the compiler complains about unused results, and it doesn't quite feel right to swallow the errors.

It does make this PR mildly risky though, because if any of these futures were quietly failing, they'll now crash the whole process (which IMO, is a more desirable outcome anyway).